templates/taxi/car_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Car</h1>
+
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>

```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Submit' one. The 'Cancel' button should lead to the previous page the user was on.